Kernel/System/DynamicField/Driver/BaseDatabase.pm

```diff
@@ -361,6 +361,12 @@ EOF
     if ( $Param{ServerError} ) {
 
         my $ErrorMessage = $Param{ErrorMessage} || 'This field is required.';
```

Comment:
Die Verwendung des Wertes einer Variable um den neuen Wert zu bestimmen ist völlig legitim. Trotzdem habe ich ein wenig gestutzt. Warum nicht in der Methode Output(), den den String direkt übergeben. 
            'Data'      => {
                'ErrorMessage'  => ($Param{ErrorMessage} || 'This field is required.'),
            }

Reply:
Gute Idee, danke dir - ich ändere es schnell.